### PR TITLE
Dark theme

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,

I have added Dark Theme and floating button to my Documentation Site.

File to be reviewed: `theme.js`

Please review and suggest

Note: It says 24 files changed. I have just added a floating icon and script tag to all the files.

### CODE REVIEW

**Review by jalajcodes — APPROVED** [review]
Everything looks good.

Just fyi instead of changing colors in javascript you could have done that using css variables
https://dev.to/ananyaneogi/create-a-dark-light-mode-switch-with-css-variables-34l8

**Comment by ApurvaSawant11** [comment]
> Everything looks good.
> 
> Just fyi instead of changing colors in javascript you could have done that using css variables https://dev.to/ananyaneogi/create-a-dark-light-mode-switch-with-css-variables-34l8

Wow! This is cleaner. Let me see what I can do about this

**Review by ArjunJunna — APPROVED** [review]
works cool. nice work.

File: js/theme.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+};
+
+const lightThemeColors = () => {
+    rootElement.style.setProperty("--theme-bg", "#ffffff");
+    rootElement.style.setProperty("--theme-text-color", "#313131");
+    rootElement.style.setProperty("--navbar-bg","#ffffff")
+    rootElement.style.setProperty("--primary","#19456b");
+    rootElement.style.setProperty("--primary-hover","#133e64");
+    rootElement.style.setProperty("--secondary","#16c79a");
+    rootElement.style.setProperty("--secondary-hover","#0eb98f");
```
Comment (animeshgarai09) [inline js/theme.js:27]: This much of setup was unnecessary for toggling themes. A better way would be to define the whole theme already.

```
.light-theme{
    --primary: #fff;
}
.dark-theme{
    --primary:#000;
}
```
Now toggle the class on the body depending on the button click.

**Review by animeshgarai09 — COMMENTED** [review]
The rest looks good. Such a clean UI. good job 😃

File: js/theme.js (hunk 1 above)
Comment (ApurvaSawant11) [reply js/theme.js:27]: I tried that. But the dark theme colors are not getting applied. Need more time to debug. So for time being, I am moving forward with this approach. 
Thanks for the review. :)

**Comment by ApurvaSawant11** [comment]
> Everything looks good.
> 
> Just fyi instead of changing colors in javascript you could have done that using css variables https://dev.to/ananyaneogi/create-a-dark-light-mode-switch-with-css-variables-34l8

Will definitely implement this approach.

**Pull request merged**